scripts/Damilare-Olagbende.js

```diff
@@ -0,0 +1 @@
+console.log("Hello World, this is Damilare Richard Olagbende with HNGi ID: HNG-01798  using js for stage 2 task")
```

Comment:
remove `:` after `ID` and replace `js` with `Javascript` plus consider making your work unique by using variables and/or functions